iothub/device/tests/IotHubModuleClientTests.cs

```diff
@@ -86,14 +87,14 @@ public void ModuleClient_CreateFromConnectionStringWithClientOptions_DoesNotThro
             };
 
             // act
-            using var moduleClient = new IotHubModuleClient(FakeConnectionString, clientOptions);
+            await using var moduleClient = new IotHubModuleClient(FakeConnectionString, clientOptions);
         }
 
         [TestMethod]
         public async Task ModuleClient_SetReceiveCallbackAsync_SetCallback_Mqtt()
         {
             var options = new IotHubClientOptions(new IotHubClientMqttSettings());
-            using var moduleClient = new IotHubModuleClient(FakeConnectionString, options);
+            await using var moduleClient = new IotHubModuleClient(FakeConnectionString, options);
```

Comment:
Not an issue with this directly.
Looking at all the existing tests and samples, it appears they are short-running and the impact of this change maybe hard to observe within the scope of the test or the sample, right?

Reply:
There will be no real impact to this change. It'll still be closed and disposed, if it was before. If it wasn't opened, close will do nothing. If was opened before but not closed, this will close it, which we should have been doing anyway.

It won't make the test run measurably longer.